lib/new_relic/agent.rb

```diff
@@ -29,7 +29,7 @@ module Agent
     require 'new_relic/metric_spec'
     require 'new_relic/metric_data'
     require 'new_relic/noticed_error'
-    require 'new_relic/agent/noticible_error'
+    require 'new_relic/agent/noticeable_error'
```

Comment:
File should be renamed somewhere below

Reply:
It is! I was delighted to see that feature included.